epubmaker_postprocessing.rb

```diff
@@ -1,10 +1,11 @@
 require 'fileutils'
 require 'net/smtp'
 
-unless defined?(ENV['TRAVIS_TEST'])
+unless (ENV['TRAVIS']) == 'true'
```

Comment:
I think I'd prefer having it in the yml, just to eliminate one layer of admin, as long as it works as expected.